service-providers/network-requester/src/config/mod.rs

```diff
@@ -159,9 +164,21 @@ impl Config {
     }
 }
 
-#[derive(Debug, Default, Clone, Deserialize, PartialEq, Serialize)]
+#[derive(Debug, Clone, Deserialize, PartialEq, Serialize)]
 #[serde(default, deny_unknown_fields)]
-pub struct NetworkRequester {}
+pub struct NetworkRequester {
+    pub description: String,
```

Comment:
should description be really part of the config? imo (personal bias) it should be treated like the mixnodes'

Reply:
it's useful to stay consistent, so I will change that